ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -421,6 +423,18 @@ def _agent_evp_proxy_base_url(self) -> Optional[str]:
                 return EVP_PROXY_AGENT_BASE_PATH
         return None
 
+    def _is_gzip_supported_by_agent(self) -> bool:
+        try:
+            info = agent.info(self.tracer._agent_url)
+        except Exception:
+            return False
+
+        if info:
+            endpoints = info.get("endpoints", [])
+            if endpoints and any(EVP_PROXY_AGENT_BASE_PATH_V4 in endpoint for endpoint in endpoints):
+                return True
+        return False
+
```

Comment:
Maybe this is enough?
```suggestion
    def _is_gzip_supported_by_agent(self) -> bool:
        return self._agent_evp_proxy_base_url() == EVP_PROXY_AGENT_BASE_PATH_V4
```

Reply:
It is! I forgot to do this after the rebase